src/components/IncrementStepper/IncrementStepper.jsx

```diff
@@ -0,0 +1,72 @@
+import { React, useCallback, useState, useEffect } from "react";
+import PropTypes from "prop-types";
+import { HiPlusSm, HiMinusSm } from "react-icons/hi";
+
+export const IncrementStepper = ({
+  min,
+  max,
+  initialValue,
+  label,
+  onClick,
```

Comment:
Würde es onChange nennen, on Click impliziert das egal wo ich in der Component klicke der onClick ausgelöst wird.